### PR TITLE
Fixing namespace conflict var_relabel

### PR DESCRIPTION
**Pull request by Melkiades**
@pawelru, I added the import for {formatters} and called it in from a custom function from rcd (formatters' one is more complete).  Should I add this to staged.deps for formatters? It is a widely used function that will not change too much in the near future. Should I copy it instead so to lose the dependency?

### CODE REVIEW

**Comment by shajoezhu** [comment]
let's not do this. this function used to link all nest packages together, such a small function, we decoupled this before to have them repeated in rcd, formatters, and teal.something. i would suggest to close this pr. thanks!

**Comment by Melkiades** [comment]
> let's not do this. this function used to link all nest packages together, such a small function, we decoupled this before to have them repeated in rcd, formatters, and teal.something. i would suggest to close this pr. thanks!

We need to change the name for namespace conflict and it is used only internally so there should not be any problem. This PR is blocking scda.test

I will copy&paste ;)

File: R/radeg.R [hunk 1]
```diff
@@ -167,16 +167,12 @@ radeg <- function(adsl,
     dplyr::mutate(BNRIND = ANRIND[ABLFL == "Y"]) %>%
     dplyr::ungroup() %>%
     dplyr::mutate(ATPTN = 1) %>%
-    dplyr::mutate(DTYPE = NA) %>%
-    var_relabel(
-      STUDYID = attr(adeg$STUDYID, "label"),
-      USUBJID = attr(adeg$USUBJID, "label")
-    )
```
Comment (Melkiades) [inline R/radeg.R:174]: broken

File: NAMESPACE [hunk 2]
```diff
@@ -23,12 +23,12 @@ export(radsub)
 export(radtr)
 export(radtte)
 export(radvs)
+export(rcd_var_relabel)
```
Comment (pawelru) [inline NAMESPACE:26]: do we have to export it at all?

Comment (pawelru) [reply NAMESPACE:26]: let's remove it from exports then
There are a few more funs like this to be removed. Conceptually, this pacakge should export only data creation funs and nothing more than that.

**Review by edelarua — APPROVED** [review]
Lgtm! Thanks @Melkiades

**Pull request merged**